### ISSUE TITLE
Relic affixation not working as intended

### ISSUE THREAD

**Issue by itsyourimp** [BareMinimumStudios/relicex#5]
**Quilt Loader: 0.23.1
Mod version: 3.4.1**

_Hey there, sorry for the bother- and first off, thank you so very much for porting this gem over to 1.20.1_
Onto the issue though, when I attempt to affix a relic to a gear item as described (Equipping a head or chest item, and right clicking with the relic whilst crouching), I get the vanilla behavior of swapping the gear as opposed to any effect relating to the items being combined.

(Video confirmation)
https://github.com/PlayerEXDirectorsCut/RelicEx/assets/34633840/948ff597-db1f-4ab5-a37c-d8f3bbf5bfb1

I could be doing this wrong- maybe the item needs to be in the offhand to affix, maybe I need a specific type of item for this relic type, maybe I even need to have Relic Shards in my inventory, but as described, this feature is wholly nonfunctional, at least for me.

**Comment by pokesmells** [comment]
Hi there. The relics are affixed to armor via the _Smithing Table_. They are not right clicked onto the gear it self. Please describe where these instructions were so it may be corrected.

**Comment by itsyourimp** [comment]
https://github.com/PlayerEXDirectorsCut/RelicEx at the 2nd dot under Misc
https://modrinth.com/mod/relicex-directors-cut at the 2nd dot under Misc
https://www.curseforge.com/minecraft/mc-mods/relicex-directors-cut at the 2nd dot under Misc
thanks again for the help!

**Comment by itsyourimp** [comment]
III hate to bear bad news here, but I also attempted to use the smithing table (and the Anvil, for good measure) to similar effect.

(Video confirmation)
https://github.com/PlayerEXDirectorsCut/RelicEx/assets/34633840/a6eef427-84f7-4abc-8521-49e1373fa3bf

I think with the changes to the smithing table in 1.20 onward, the function got messed up somewhere, unless this was wholly unintended. Still, thanks for the insight

**Comment by pokesmells** [comment]
> III hate to bear bad news here, but I also attempted to use the smithing table (and the Anvil, for good measure) to similar effect.
> 
> (Video confirmation) https://github.com/PlayerEXDirectorsCut/RelicEx/assets/34633840/a6eef427-84f7-4abc-8521-49e1373fa3bf
> 
> I think with the changes to the smithing table in 1.20 onward, the function got messed up somewhere, unless this was wholly unintended. Still, thanks for the insight

Hi there. It was a major oversight on our part, as the original functionality was intended to be used in the smithing table. 

I verified by using a **1.19.2** instance, and confirmed that the relics do not count as a material to be able to be used in the new reworked smithing table,

 What we might do is rework the original feature to be reincorporated into the anvil or make it so you are able to add the relics back from the smithing table, at the moment this will not work.
 
 Now about the descriptions, we merely tidied up the original README and then pasted it with a brand new banner for the time being, so for now, that section shall be removed and then re-added when this issue is fixed.
 
 Thank you for the detailed issue reports!

**Comment by itsyourimp** [comment]
ah, the detailed stuff is no biggie, all the better to help this get solved
i wholly think it could use the reworked smithing table, maybe the relics are affixed with a template that could be found around the world, maybe they could be detachable with the grindstone at the cost of some relic shards...
ah- but that's just me musing now, hardly my place to tell you guys how to make your port. thanks again for your help poke!

**Comment by karuzumi** [comment]
Hello! I've addressed this problem in the 3.6.2+1.20.1 update, you can either grab a new jar build from releases immediately or wait for the updates to Modrinth & Curseforge, which will be updated shortly.

Thanks for reporting! Please send a response if it works for you. @itsyourimp 
See: #6

**Comment by itsyourimp** [comment]
I was briefly confused when I saw 3.**6**.2, but I got the general idea after checking the auto-update for Prism.
Works like a dream now though! And in a lovely turn of events, it ended up being compatible with the Truly Modular mod, at least where armor is concerned. I assume it just checks for any armor that fits the description of "goes on head or chest" but- I'll stop myself before i get ramble-y

![2024-01-29_09 21 20](https://github.com/PlayerEXDirectorsCut/RelicEx/assets/34633840/60c87967-5f41-4dcc-b014-d8c5e868624f)
![2024-01-29_09 21 20_2](https://github.com/PlayerEXDirectorsCut/RelicEx/assets/34633840/4bd44295-60cd-4dcf-9b2b-adbfc315d800)

Thanks for the aid, insight, and fix! Good luck with whatever you do next!
